lib/ansible/plugins/callback/logstash.py

```diff
@@ -87,33 +103,72 @@ def __init__(self):
             self.disabled = True
             self._display.warning("The required python-logstash is not installed. "
                                   "pip install python-logstash")
-        else:
+
+        self.start_time = datetime.utcnow()
+
+    def _init_plugin(self):
+        if not self.disabled:
```

Comment:
Is this necessary since the task_queue_manager checks it?
https://github.com/ansible/ansible/blob/devel/lib/ansible/executor/task_queue_manager.py#L350.